src/Symfony/Component/HttpKernel/Tests/KernelTest.php

```diff
@@ -64,6 +64,19 @@ public function testClone()
         $this->assertNull($clone->getContainer());
     }
 
+    /**
+     * @expectedException \InvalidArgumentException
```

Comment:
can you check the message too ?

Reply:
Sure, will append a commit in a few minutes.

Reply:
Done.

Reply:
See 415254a412aa3387a11cd9901b4ab32efa443679